src/components/shared/eventcard/eventcard.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Day from 'components/shared/day';
+import Box from 'components/shared/box';
+import Text from 'components/shared/text';
+import Button from 'components/shared/button';
+
+import EventPlaceholderImage from 'images/event-placeholder.png';
+
+import ClockIcon from 'images/clock.svg';
+import MapMarkerIcon from 'images/map-marker.svg';
+
+import {
+  Article,
+  ImageWrapper,
+  Image,
+  RSVPBox,
+  IconList,
+  IconRow,
+  IconText,
+  IconLink,
+} from './eventcard.css';
+
+import { parse, format } from 'date-fns';
+
+const EventCard = ({ event }) => {
+  const { name, local_date, time, featured_photo, venue, short_link } = event;
+
+  const image = () => {
+    if (featured_photo) {
+      return <Image src={featured_photo.photo_link} alt="event" />;
+    } else {
+      return <Image src={EventPlaceholderImage} alt="event" />;
+    }
+  };
+
+  const parsedDateTime = parse(time, 'T', new Date());
+  const formattedTime = format(parsedDateTime, 'h:mm a');
+
+  const venueLink = () => {
```

Comment:
Again no need for this to be inline! Can just be a standalone component and receive values via props rather than via closure.

Reply:
Yep. Done in e3e1f9b.